### ISSUE TITLE
What information is contained in the part_offsets tensor output?

### ISSUE THREAD

**Issue by jtrammell-dla** [tensorflow/tfjs#4591]
I'm trying to finetune the bodypix model and I cannot for the life of me figure out how to generate the targets for the part_offsets output. Based on its shape ([H,W,48]) I presume it is a pair of numbers for each of the 24 body parts. Furthering that assumption, and the name, an X/Y offset? But it's clamped to 0->1. And body parts are regions, not points, so an offset for a bodypart makes no sense to me. Can you clear this up a little for me?

**Comment by pyu10055** [comment]
@tylerzhu-github Hi Tyler can you help to comment on this question? Thanks.

**Comment by tylerzhu-github** [comment]
Hi @jtrammell-dla,

Thanks for your inquiry! 

That's a very good questions. The output node is reserved for future use cases.

The best way is to not set any target for that head and ignore it in the total loss.

Best,
Tyler

**Comment by jtrammell-dla** [comment]
Ah, so!!! That explains it, thank you! Now I'm curious, though. Can you tell us what sort of future use cases you had in mind? And thank you for the quick response.

**Comment by rthadur** [comment]
Please check latest body segmentation model for latest bodypix model demo https://github.com/tensorflow/tfjs-models/tree/master/body-segmentation
